src/Controls/src/Core/HandlerImpl/Label/Label.cs

```diff
@@ -33,6 +33,10 @@ internal static new void RemapForControls()
 			// And we map some of the other property handlers to Controls-specific versions that avoid stepping on HTML text settings
 
 			LabelHandler.Mapper = ControlsLabelMapper;
+
+#if ANDROID
+			LabelHandler.CommandMapper.AppendToMapping<Label, ILabelHandler>(nameof(Frame), MapFrame);
+#endif
```

Comment:
Just a new mapper - Android-only for now, but the same will happen for iOS and Windows.